demo-project/src/main/java/com/example/demoproject/DemoController.java

```diff
@@ -8,15 +9,17 @@
 import org.jooq.Configuration;
 import org.jooq.DSLContext;
 import org.jooq.impl.DSL;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
 
 @RestController
 public class DemoController {
     private final DSLContext ctx;
-    private final Function<Configuration,DemoRepository> repository;
+    private final Function<Configuration, DemoRepository> repository;
```

Comment:
You generally shouldn't include line changes that only change formatting, unless the intent of the commit is a change in formatting style. Changes like those make it very hard to see what functional changes the commit contains.